draft-ietf-ipsecme-ikev2-pqc-auth.md

```diff
@@ -136,6 +136,8 @@ When generating a signature with a PQC signature algorithm, the IKEv2 implementa
 
 When verifying a signature with a PQC signature algorithm, the IKEv2 implementation takes the InitiatorSignedOctets string or the ResponderSignedOctets string (as appropriate), logically sends it to the identity hash (which leaves it unchanged), and then passes it into the PQC signature verifier as the message to be verified (with empty context string, if applicable).
 
+IKEv2 peers supporting the PQC authentication mechanism defined in this specification MUST implement IKEv2 message fragmentation as specified in {{!RFC7383}}, since PQC public keys and signatures can be significantly larger than those used in traditional algorithms. For example, ML-DSA-44 requires a public key of 1,312 bytes and a signature of 2,420 bytes, while even the smallest SLH-DSA signature is around 7,856 bytes. As guidance, IKEv2 peers should assume a minimum PMTU of 1280 bytes for IPv6 (per {{?RFC8200}}) and, where legacy IPv4 networks are a consideration, an effective MTU of 576 bytes for IPv4 (per {{?RFC1122}}).
```

Comment:
I believe that "SHOULD" is enough with a clarification that this is not relevant if IP fragmentation is a not an issue (e.g. reliable transport is used for IKE or underlying network supports larger MTUs). Something along the lines.

Reply:
Good point, updated text.

Reply:
Hmm... It seems that the updated text is commited directly to the "main", I can see it there and do not see it in the PR. Anyway, the new text is OK with me. I approve the PR in anticipating that this confusion is resolved and the updated text is used when PR is meged.